src/viz/layer/Layer.ts

```diff
@@ -688,19 +700,19 @@ export class Layer extends WithEvents implements StyledLayer {
    * Manage data state and asociated events
    */
   private _sendDataEvent(referer: 'onViewportLoad' | 'onAfterRender') {
-    const isGeoJsonLayer = this._source.sourceType === 'GeoJSON';
+    const isGeoJsonLayer = this._source.sourceType === 'GeoJSONSource';
```

Comment:
After my last merge, this is 'GeoJSON' not 'GeoJSONSource' (maybe it's not the best move, but I did it and it won't work otherwise...)

Reply:
Done